clusterloader2/testing/l4ilb/config.yaml

```diff
@@ -117,6 +120,31 @@ steps:
     Method: WaitForControlledPodsRunning
     Params:
       action: gather
+{{if $EXEC_COMMAND}}
+- name: Starting measurements for nodepool upgrade
```

Comment:
This test currently uses service.yaml which tests the default TrafficPolicy (Cluster). It would be great to have the nodepool upgrade scenario verified for "Local" traffic policy too. The service.yaml traffic policy field can be parameterized like in https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/testing/l4lb/service.yaml.

This can be a follow up.

Reply:
Yes, I will make the changes in the next PR.
Thank you for the comments.